.readthedocs.yml

```diff
@@ -0,0 +1,17 @@
+version: 2
+
+sphinx:
+  configuration: docs/source/conf.py
+
+formats:
+  - pdf
+
+sphinx:
+  fail_on_warning: true
+
+python:
+  version: 3.7
+  install:
+    - requirements: docs/source/requirements.txt
```

Comment:
should we use this workaround to avoid an additional requirements.txt? https://github.com/readthedocs/readthedocs.org/issues/4912#issuecomment-664002569

Reply:
looking into this right now, as I need to install docs locally anyways

Reply:
Yes agreed - this would be nicer.